core/controllers/creator_dashboard.py

```diff
@@ -287,7 +271,6 @@ def get(self):
 class ExplorationDashboardStatsHandler(base.BaseHandler):
     """Returns the most recent open feedback for an exploration."""
 
-    @base.require_fully_signed_up
     def get(self, exploration_id):
```

Comment:
Is it used anymore? I searched for the url and I don't see it requested anywhere.